packages/svelteui-core/src/components/Textarea/Textarea.stories.svelte

```diff
@@ -0,0 +1,50 @@
+<script lang="ts">
+	import { Meta, Story, Template } from '@storybook/addon-svelte-csf';
+	import { Textarea } from './index';
+</script>
+
+<Meta title="Components/Textarea" component={Textarea} />
+
+<Template let:args>
+	<Textarea {...args} />
+</Template>
+
+<Story
+	name="Default"
+	id="TextareaStory"
+	args={{
+		label: 'Your story',
+		placeholder: 'Once upon a time'
+	}}
+/>
+
+<Story
+	name="Description"
+	id="TextareaDescriptionStory"
+	args={{
+		label: 'Your story',
+		description: 'Tell us about yourself',
+		placeholder: 'Once upon a time'
+	}}
+/>
+
+<Story
+	name="Resizeable"
+	id="TextareaResizeableStory"
+	args={{
+		label: 'Your story',
+		placeholder: 'Once upon a time',
+		rows: 4,
```

Comment:
I don't think this is being supported, since it's not declared as one of the props in Textarea nor Input. Is this something to be implemented later?

Reply:
You mean `rows`? It's one of `<textarea />`'s attributes https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea#attributes

Reply:
You're right. It's not working. probably not being passed through correctly.

Reply:
Hum, yah, the problem is with the demo configurator. I'll open a ticket to fix it, but it's not a blocker for this PR